### PR TITLE
curvefs: add global cache client like memcached

### PR DESCRIPTION
**Pull request by fansehep**
Signed-off-by: fan <yfan3763@gmail.com>

### What problem does this PR solve?

Issue Number: #xxx 

Problem Summary:

### What is changed and how it works?

What's Changed:

How it Works:

Side effects(Breaking backward compatibility? Performance regression?):

### Check List

- [ ] Relevant documentation/comments is changed or added
- [ ] I acknowledge that all my contributions will be made under the project's license

### CODE REVIEW

File: curvefs/test/client/client_memcache_test.cpp [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+//         manager_.Uninit();
+//         auto str = absl::StrCat("kill -9 ", memcached_pid);
+//         ::system(str.c_str());
+//         std::this_thread::sleep_for(std::chrono::seconds(3));
+//     }
+//     pid_t memcached_pid;
+//     KvClientManager manager_;
+// };
+
+// TEST_F(MemCachedTest, MultiThreadGetSetTest) {
```
Comment (ilixiaocui) [inline curvefs/test/client/client_memcache_test.cpp:78]: TEST(MemCachedTest, DISABLED_MultiThreadGetSetTest) { ... }

**Comment by ilixiaocui** [comment]
recheck

**Comment by ilixiaocui** [comment]
recheck

**Comment by ilixiaocui** [comment]
recheck

**Comment by ilixiaocui** [comment]
recheck

**Comment by ilixiaocui** [comment]
recheck

**Comment by fansehep** [comment]
> recheck

Congratulations! 👍🏻

**Pull request merged**